src/ext/oblivc/obliv_bits.c

```diff
@@ -67,123 +77,114 @@ typedef struct tcp2PTransport
   bool needFlush;
   bool keepAlive;
   int sinceFlush;
-#ifdef PROFILE_NETWORK
   size_t bytes;
-  int flushCount;
+  size_t flushCount;
   struct tcp2PTransport* parent;
-#endif
 } tcp2PTransport;
 
-size_t tcp2PBytesSent(ProtocolDesc* pd) 
-#ifdef PROFILE_NETWORK
-  { return ((tcp2PTransport*)(pd->trans))->bytes; }
-#else
-  { return 0; }
-#endif
-
-int tcp2PFlushCount(ProtocolDesc* pd)
-#ifdef PROFILE_NETWORK
-  { return ((tcp2PTransport*)(pd->trans))->flushCount; }
-#else
-  { return 0; }
-#endif
+// Profiling output
+size_t tcp2PBytesSent(ProtocolDesc* pd) { return ((tcp2PTransport*)(pd->trans))->bytes; }
+size_t tcp2PFlushCount(ProtocolDesc* pd) { return ((tcp2PTransport*)(pd->trans))->flushCount; }
 
 static int tcp2PSend(ProtocolTransport* pt,int dest,const void* s,size_t n)
-{ struct tcp2PTransport* tcpt = CAST(pt);
+{ 
+  struct tcp2PTransport* tcpt = CAST(pt);
   size_t n2=0;
   tcpt->needFlush=true;
-  /*while(n>n2) {*/
-	/*int res = write(tcpt->sock,n2+(char*)s,n-n2);*/
-	/*if(res<=0) { perror("TCP write error: "); return res; }*/
-	/*n2+=res;*/
-/*#ifdef PROFILE_NETWORK*/
-	/*tcpt->bytes += res;*/
-/*#endif*/
-  /*}*/
   while(n>n2) {
-	int res = fwrite(n2+(char*)s,1,n-n2,tcpt->sockStream);
-	if(res<0) { perror("TCP write error: "); return res; }
-	n2+=res;
-#ifdef PROFILE_NETWORK
-	tcpt->bytes += res;
-#endif
+	  int res = fwrite(n2+(char*)s,1,n-n2,tcpt->sockStream);
+	  if(res<0) { perror("TCP write error: "); return res; }
+	  n2+=res;
   }
   return n2;
 }
 
+static int tcp2PSendProfiled(ProtocolTransport* pt,int dest,const void* s,size_t n)
+{
+  struct tcp2PTransport* tcpt = CAST(pt);
+  size_t res = tcp2PSend(pt, dest, s, n);
+  if (res >= 0) tcpt->bytes += res;
+  return res;
+}
+
 static int tcp2PRecv(ProtocolTransport* pt,int src,void* s,size_t n)
-{ int res=0,n2=0;
+{ 
+  int res=0,n2=0;
 	struct tcp2PTransport* tcpt = CAST(pt);
 	if(tcpt->needFlush)
 	{
 		fflush(tcpt->sockStream);	
-#ifdef PROFILE_NETWORK
-                tcpt->flushCount++;
-#endif
 		tcpt->needFlush=false;
 	}
-  /*while(n>n2)*/
-  /*{ res = read(((tcp2PTransport*)pt)->sock,n2+(char*)s,n-n2);*/
-	/*if(res<=0) { perror("TCP read error: "); return res; }*/
-	/*n2+=res;*/
-  /*}*/
   while(n>n2)
-  { res = fread(n2+(char*)s,1,n-n2, tcpt->sockStream);
-	if(res<0 || feof(tcpt->sockStream)) { perror("TCP read error: "); return res; }
-	n2+=res;
+  { 
+    res = fread(n2+(char*)s,1,n-n2, tcpt->sockStream);
+	  if(res<0 || feof(tcpt->sockStream)) { perror("TCP read error: "); return res; }
```

Comment:
I would like to state, for the record, that although I am a proponent of tabs, I'm an even stronger proponent of consistency. Fixed this below.